### PR TITLE
✨ Add cloud functions support

### LINKED ISSUE #7: (Config) Create cloud functions repo in Github

### PR DESCRIPTION
**Pull request by enriquepalaciosv**
Adds boilerplate code for pixar image generation and rewards triggers

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -19,5 +19,5 @@ jobs:
 
       - name: Run lint
         run: |
-          yarn install --frozen-lockfile
+          yarn install
```
Comment (victorstein) [inline .github/workflows/lint.yml]: Seems like this was pushed as a fix for linting, which does not seem right. Is there any other reason for this or can we revert?

```suggestion
          yarn install --frozen-lockfile
```

**Review by victorstein — CHANGES_REQUESTED** [review]
Thanks for integrating this to the repo @enriquepalaciosv! All CI checks should be passing now. I just left one comment behind then I'm ready to approve.

# Great Job
gif

File: .github/workflows/lint.yml (hunk 1 above)
Comment (enriquepalaciosv) [reply .github/workflows/lint.yml]: Let's revert it. Thanks for checking this out.

**Review by alvarocardenasc — APPROVED** [review]

File: .github/workflows/lint.yml (hunk 1 above)
Comment (victorstein) [reply .github/workflows/lint.yml]: resolved on the latest commit

**Review by victorstein — APPROVED** [review]

**Pull request merged**